src/mantine-hooks/src/use-id/use-id.test.ts

```diff
@@ -13,7 +13,7 @@ describe('@mantine/hooks/use-id', () => {
   });
 
   it('returns generated id if id was not provided', () => {
-    const hook = renderHook(() => useId(null, () => 'test-id'));
+    const hook = renderHook(() => useId(undefined, () => 'test-id'));
```

Comment:
Please revert this change and do not change files apart from your pr scope

Reply:
Okay but there was a typescript error.
changing it to undefined fixed it